elasticsearch/tests/elasticsearch_test.py

```diff
@@ -84,8 +84,10 @@ def test_defaults():
     assert c["volumeMounts"][0]["mountPath"] == "/usr/share/elasticsearch/data"
     assert c["volumeMounts"][0]["name"] == uname
 
+    # volumeClaimTemplates
     v = r["statefulset"][uname]["spec"]["volumeClaimTemplates"][0]
     assert v["metadata"]["name"] == uname
+    assert v["metadata"]["labels"] == {}
```

Comment:
`spec.volumeClaimTemplates.metadata.labels` won't exist in the generated template with default values (if )`persistence.labels.enabled=false`).

```suggestion
    assert "labels" not in v["metadata"]
```